### PR TITLE
Add vue credential repository.

### PR DESCRIPTION
**Pull request by dlongley**
This new credential repository also uses a mongo database to store credentials when the browser used is Safari (or a webkit-based browser that supports the Storage Access API). This is required because of forced third party cookie partitioning when multiple levels of iframes are used.

The entire credential demo works on Safari w/o any setting changes when using this new repository and the latest authorization.io#vue branch for the mediator. We'll need to test on iOS once we merge the related PRs and deploy. The same fixes should (in theory) work for the web payments polyfill as well.

### CODE REVIEW

File: vue-credential-repository/components/CredentialCard.vue [hunk 1]
```diff
@@ -0,0 +1,206 @@
+<template>
+  <div class="row" style="margin: 0;">
+    <div class="br-card-id-1 br-card-id-1-border" xstyle="cardStyle">
```
Comment (davidlehn) [inline vue-credential-repository/components/CredentialCard.vue]: xstyle?

File: angular-credential-issuer/lib/config.js [hunk 2]
```diff
@@ -25,5 +25,6 @@ config.views.system.packages.push({
 
 // URLs for Authorization.io
 cc('views.vars.authorization-io.baseUri', () =>
-  'https://demo.authorization.io');
+  'https://authorization.localhost:33443');
+//  'https://demo.authorization.io');
```
Comment (davidlehn) [inline angular-credential-issuer/lib/config.js]: angular code only works with localhost now?

File: vue-credential-repository/lib/config.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+// common paths
+config.paths.cache = path.resolve(path.join(rootPath, '.cache'));
+config.paths.log = path.join(
+  os.tmpdir(), 'example.credential-repository.localhost');
+
+// URLs for Authorization.io
+cc('views.vars.authorization-io.baseUri', () =>
+  'https://authorization.localhost:33443');
+//  'https://demo.authorization.io');
```
Comment (davidlehn) [inline vue-credential-repository/lib/config.js]: Another commented host to flip before merging.

**Comment by dlongley** [comment]
@davidlehn, all your comments addressed.

**Pull request merged**